Readme.md

```diff
@@ -49,3 +49,21 @@ Directory `samples/` contains sample requests for each `X-GitHub-Event`
 # Deploy
 
 `git push heroku master`
+
+# Docker
+
+## Start the environment
+
+```
+eval "$(docker-machine env dev)"
```

Comment:
You need to have a `docker-machine` called 'dev' created, right? Maybe add how to do this to the README? 🤔